src/main/java/com/rockset/jdbc/RocksetTable.java

```diff
@@ -45,7 +49,40 @@ public RocksetTable(String catalog, String schema, String tableNamePattern,
     incol.add(new Column("type", Column.ColumnTypes.STRING));
     incol.add(new Column("occurrences", Column.ColumnTypes.NUMBER));
     incol.add(new Column("total", Column.ColumnTypes.NUMBER));
-    this.describe = new RocksetResultSet(incol, response.getResults());
+	
+	//Skip over the field description row with column of null type where the same column has a valid type
+    //As of now moving all the null type rows to the last.
+    List<Object> fieldsDescription= response.getResults();
+
+    //Get all non null type rows
+    List<Object>fieldsDescriptionWithTypeAsNonNull = fieldsDescription.stream()
+    .filter(fieldName -> {
+      try {
+        return !mapper.readTree(mapper.writeValueAsString(fieldName)).get("type").asText().equals("null");
+      } catch (IOException e) {
+        e.printStackTrace();
+      }
+      return false;
+    })
+    .collect(Collectors.toList());
+
+    //Get all null type rows
+    List<Object>fieldsDescriptionWithTypeAsNull = fieldsDescription.stream()
+            .filter(fieldName -> {
+              try {
+                return mapper.readTree(mapper.writeValueAsString(fieldName)).get("type").asText().equals("null");
+              } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
use `log` to print the stack trace. Same for above exception. Also is it fine to ignore the exception or do we need to propagate it to the user?

Reply:
No need to propagate it to the user. The recordset already getting dumped in the old code itself. So I feel just handle the exception is fine and I will remove the printStackTrace.